thirdparty/cmake_modules/koreader_thirdparty_common.cmake

```diff
@@ -3,6 +3,12 @@ if(NOT DEFINED PROCESSOR_COUNT)
     ProcessorCount(N)
     # 0 if unknown
     set(PROCESSOR_COUNT ${N})
+
+    # Some compilations (like harfbuzz) are known to OOM on memory-limited CI.
+    set(CIRCLECI_PROCESSOR_COUNT ${N})
+    if($ENV{CIRCLECI})
+        set(CIRCLECI_PROCESSOR_COUNT 1)
+    endif()
```

Comment:
@poire-z @NiLuJe This is the basic principle of what we need but I'm not completely fond of the form I gave it. Any thoughts?

Reply:
May be like this ? (with another better word than "light" ? limited, restricted, friendly ?)
```
    set(PROCESSOR_COUNT ${N})
    set(LIGHT_PROCESSOR_COUNT ${N})

    # Some compilations (like harfbuzz) are known to OOM on memory-limited CI.
    if($ENV{CIRCLECI})
        set(LIGHT_PROCESSOR_COUNT 1)
    endif()
```

Reply:
Something like LOW_MEM?

Reply:
Why not. (Although "light" feels more generic, in case we need it in situations others than low memory - dunno if any other could happen though.)

Reply:
constrained?